modules/tensor_mechanics/src/materials/ComputePFFractureStressBase.C

```diff
@@ -42,6 +46,7 @@ ComputePFFractureStressBase::ComputePFFractureStressBase(const InputParameters &
     _gc(getMaterialProperty<Real>("gc_prop")),
     _pressure(getDefaultMaterialProperty<Real>("fracture_pressure")),
     _use_current_hist(getParam<bool>("use_current_history_variable")),
+    _use_snes_vi_solver(getParam<bool>("use_snes_vi_solver")),
     _H(declareProperty<Real>("hist")),
     _H_old(getMaterialPropertyOld<Real>("hist")),
```

Comment:
I think it makes more sense for _H_old to be a pointer. We really don't need _H_old if either _use_current_hist or _use_snes_vi_solver is true. But maybe this should go to another PR.

Reply:
Definitely another PR :-)